sdk/src/androidTest/java/com/karhoo/sdk/api/LoyaltyConversionIntregrationTest.kt

```diff
@@ -0,0 +1,229 @@
+package com.karhoo.sdk.api
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.tomakehurst.wiremock.junit.WireMockRule
+import org.assertj.core.api.Java6Assertions.assertThat
+import com.karhoo.sdk.api.model.LoyaltyConversion
+import com.karhoo.sdk.api.network.response.Resource
+import com.karhoo.sdk.api.testrunner.SDKTestConfig
+import com.karhoo.sdk.api.util.ServerRobot.Companion.EMPTY
+import com.karhoo.sdk.api.util.ServerRobot.Companion.GENERAL_ERROR
+import com.karhoo.sdk.api.util.ServerRobot.Companion.INVALID_JSON
+import com.karhoo.sdk.api.util.ServerRobot.Companion.LOYALTY_CONVERSION
+import com.karhoo.sdk.api.util.ServerRobot.Companion.LOYALTY_ID
+import com.karhoo.sdk.api.util.ServerRobot.Companion.NO_BODY
+import com.karhoo.sdk.api.util.serverRobot
+import org.junit.After
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.net.HttpURLConnection.HTTP_OK
+import java.net.HttpURLConnection.HTTP_UNAUTHORIZED
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(AndroidJUnit4::class)
+@Ignore
+class LoyaltyConversionIntregrationTest {
+
+    @get:Rule
+    var wireMockRule = WireMockRule(SDKTestConfig.wireMockOptions)
+
+    private val latch = CountDownLatch(1)
+
+    @Before
+    fun setUp() {
+        serverRobot {
+            successfulToken()
+        }
+    }
+
+    @After
+    fun tearDown() {
+        wireMockRule.resetAll()
+    }
+
+    /**
+     * Given:   Cancellation fee on a trip is requested
```

Comment:
Is the javadoc correct for these tests? Should it be referring to loyalty conversion?

Reply:
Ah yes, missed that